src/test/onboarding.e2e.ts

```diff
@@ -0,0 +1,108 @@
+import { test, expect, ElectronApplication, Page } from '@playwright/test';
+import { _electron as electron } from 'playwright';
+import { ipcMainEmit } from 'electron-playwright-helpers';
+
+import AccessResponseFixtures from './fixtures/AccessResponse.json';
+
+const transitionToTakePalce = async (ms: number): Promise<void> => {
```

Comment:
typo here: `transitionToTakePlace`. A better, more descriptive name could be `wait`